### PR TITLE
update readme, add security config and endpoints

### PR DESCRIPTION
**Pull request by ThePapiok**

### CODE REVIEW

**Review by DarthRevan77797 — APPROVED** [review]
Works as intended
Tested:
logging as admin and user works
when not authenticated can't access /admin     ok
when authenticated as user can't access /admin     ok
logout work etc.
login form show up correctly etc.

File: src/main/java/umk/mat/pajda/ProjektZespolowy/configs/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+@Configuration
+public class SecurityConfig {
+
+  @Bean
+  public UserDetailsService userDetailsService() {
+    UserDetails user =
+        User.builder()
+            .username("user")
+            .password(passwordEncoder().encode("user"))
```
Comment (kacper-cf) [inline src/main/java/umk/mat/pajda/ProjektZespolowy/configs/SecurityConfig.java:24]: shouldn't this be more random string hidden in env variables? 🤔

File: src/main/java/umk/mat/pajda/ProjektZespolowy/configs/SecurityConfig.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            .roles("USER")
+            .build();
+
+    UserDetails admin =
+        User.builder()
+            .username("admin")
+            .password(passwordEncoder().encode("admin"))
```
[the remark this answers is not in the document]
Comment (ThePapiok) [reply src/main/java/umk/mat/pajda/ProjektZespolowy/configs/SecurityConfig.java:31]: i added it: https://github.com/Programowanie-Zespolowe-PAJDA/Backend/pull/15

File: src/main/java/umk/mat/pajda/ProjektZespolowy/configs/SecurityConfig.java (hunk 1 above)
Comment (ThePapiok) [reply src/main/java/umk/mat/pajda/ProjektZespolowy/configs/SecurityConfig.java:24]: i added it: https://github.com/Programowanie-Zespolowe-PAJDA/Backend/pull/15

**Pull request merged**